docs/api.md

```diff
@@ -484,9 +484,11 @@ JSON post body can be sent in the following format:
 ```
 
 > **NOTE**: logs sent to Loki for every stream must be in timestamp-ascending
-> order, meaning each log line must be more recent than the one last received.
-> If logs do not follow this order, Loki will reject the log with an out of
-> order error.
+> order, meaning each log line must be more equal or greater to the line
```

Comment:
`more equal`

also this won't be true, it's equal or greater unless the content is the same then it has to be greater

Reply:
the new rules are confusing :) I'd like to just simplify this section and have it refer to the main rules for more details. WDYT?

Reply:
maybe something like `logs sent to Loki for every stream must be in timestamp-ascending order, log lines with identical timestamps are only allowed if their content differs.  If a log is received with a timestamp older than the last log received it is rejected with an out of order error.  If a log is received with the exact timestamp and content as the last log received it is silently ignored.  refer to the ...